piksi_multi_cpp/src/sbp_callback_handler/sbp_callback_handler_factory.cc

```diff
@@ -0,0 +1,23 @@
+#include <libsbp/system.h>
+#include <ros/console.h>
+#include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler.h"
+#include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_factory.h"
+#include "piksi_multi_cpp/sbp_callback_handler/sbp_callback_handler_heartbeat.h"
+
+namespace piksi_multi_cpp {
+
+SBPCallbackHandler::SBPCallbackHandlerPtr
+SBPCallbackHandlerFactory::createSBPRelayCallbackBySBPMsgType(
```

Comment:
maybe createRealyBySBPMsgType is enough, as the rest is already in the type name